requirements.txt

```diff
@@ -1,33 +1,14 @@
-astroplan
-astropy >= 3.0.0
-ccdproc
-codecov
-coveralls
-dateparser
+# These are packages that are NOT available via our
+# choice of conda repositories; there are less well
+# known repositories that do offer them, but that is
+# more of a security risk.
+
+# Python wrapper for ffmpeg.
 ffmpy
-gcloud
-google-cloud-storage
-matplotlib >= 2.0.0
-mocket
-numpy >= 1.12.1
-pycodestyle == 2.3.1
-pymongo >= 3.2.2
-pyserial >= 3.1.1
-pytest >= 3.4.0
-python_dateutil >= 2.5.3
-PyYAML >= 3.11
-pyzmq >= 15.3.0
-readline
-requests
-scikit_image >= 0.12.3
-scipy >= 0.17.1
 
-# Jupyter-notebook requires Tornado versions between 4 and 5. See:
-# https://stackoverflow.com/questions/49141525/install-jupyter-notebook-on-miniconda
-# https://github.com/ipython/ipython/issues/8249
-# https://github.com/jupyter/help/issues/324
+# Mock object support for Python testing.
+mocket
 
-tornado == 4.5.3
+# State machine library for Python.
 transitions >= 0.4.0
```

Comment:
And technically I'm still a maintainer on this project though I haven't done anything for years. :confused: I can find out if there is any plan.